src/Database/Driver/Mysql.php

```diff
@@ -185,4 +190,15 @@ public function supportsNativeJson(): bool
 
         return $this->_supportsNativeJson = version_compare($this->_version, '5.7.0', '>=');
     }
+
+    /**
+     * Returns the maximum alias length allowed.
+     * This can be different than the maximum identifier length for columns.
+     *
+     * @return int|null Maximum alias length or null if no limit
+     */
+    public function getMaxAliasLength(): ?int
```

Comment:
This exact same code is still being repeated in 3 classes. I guess it would be better to define the constant also in `Driver` and just override it's value in child classes and make `Driver::getMaxAliasLength()` return the constant value.

Reply:
Is that ok for user-defined drivers? They wouldn't know to override the constant so we'd have to avoid using `MAX_ALIAS_LENGTH` outside of the `getMaxAliasLength()` wrapper, right?

I don't mind it.  Just want to make sure I don't introduce anything too painful.

Reply:
Even with currently implementation any custom drivers extending `Driver` will have their `getMaxAliasLength()`. So overriding the constant is no worse than overriding the method.